src/models/Conversation.js

```diff
@@ -0,0 +1,42 @@
+import {Record, List} from 'immutable';
+import Message from './Message';
+
+const ConversationRecord = Record({
+    id: null,
+    userToken: '',
+    messages : new List(),
+    userId: null
+});
+
+export default class Conversation extends ConversationRecord {
+
+    static mapper (conversation) {
+        let conver = new Conversation(conversation);
+        conver = conver.set('userId', conversation.id);
+        if (conversation.messages) {
+            conver = conver.set('messages', conver.addMessage(conversation.messages));
+        }
+        return conver;
+    }
+
+    addMessage (messages) {
+        let newMessages = new List();
```

Comment:
For det api'et til det messenger pakken vi bruker ikke er lag vår datastruktur, og det fant jeg ut etter jeg hadde skrevet alt annet. Så fant ut at det var enklere å gjøre dette enn å endre alt annet.